docs/examples/go-ipfs-as-a-library/main.go

```diff
@@ -57,6 +57,10 @@ func createTempRepo(ctx context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	
+	// Add your custom configuration to the repository config
+	// For example, you can enable sharding with:
+	// cfg.Experimental.ShardingEnabled = true
```

Comment:
This tutorial is designed to be a gentle introduction.

It would be better to have a second tutorial on how to enable experimental features on go-ipfs

Reply:
Just gave it a full fledged markdown document!